parallel-install/pkg/prerequisites/prerequisites.go

```diff
@@ -1,8 +1,13 @@
+//Package prerequisites implements logic preparing the cluster for Kyma installation.
```

Comment:
```suggestion
//"Package prerequisites" implements the logic preparing the cluster for Kyma installation.
```

Reply:
Applied without double quotes. These are not used here by the go doc convention.